tests/components/multiselect/multiselect.test.jsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {mountWithIntl} from 'tests/helpers/intl-test-helper.jsx';
+import MultiSelect from 'components/multiselect/multiselect.jsx';
+
+describe('components/multiselect/multiselect', () => {
+    const totalCount = 8;
+    const optionsNumber = 8;
+    const users = [];
+    for (var i = 0; i < optionsNumber; i++) {
+        users.push({id: i, value: i});
+    }
+
+    const baseProps = {
+        users,
+        totalCount,
+        values: [{id: 100, value: 100}],
+        saving: false,
+        options: users,
+        perPage: 5,
+    };
+
+    test('should match snapshot', () => {
+        const wrapper = mountWithIntl(
```

Comment:
Why `mountWithIntl` vs. just `shallow`? The snapshot, as is, seems useless to me: I have no way to verify that this is correct now, much less if a future change meant to break it.

Reply:
Ah, hmm. Is the `ref` thing hard to fix to use `shallow`? I agree that the snapshots /do/ effectively assert the behaviour in question, but I'm worried that it's buried.

0/5 on fixing it, especially given that this is a patch. It might be a broader discussion as to the effectiveness of snapshots in general.